src/atomate2/vasp/schemas/calculation.py

```diff
@@ -443,6 +449,48 @@ def from_vasp_outputs(
             magmoms = [m["tot"] for m in outcar.magnetization]
             structure.add_site_property("magmom", magmoms)
 
+        # Parse DOS properties
+        if hasattr(vasprun, "complete_dos"):
```

Comment:
Please could you refactor this to a separate function, just to keep `from_vasp_outputs` from growing too large.

Reply:
Certainly. That makes sense! I will make that change when I write up the unit tests sometime in the next few days, now that the pymatgen version has been updated.

Reply:
Resolved in e1b229d.